static/js/containers/UserPage_test.js

```diff
@@ -552,6 +552,29 @@ describe("UserPage", function() {
           });
         });
       });
+
+      it('should disable the save button while a PATCH is in progress', () => {
+        const username = SETTINGS.user.username;
+        return renderComponent(`/learner/${username}`, userActions).then(([, div]) => {
+          let addButton = div.querySelector(".add-education-button");
+
+          TestUtils.Simulate.click(addButton);
+
+          let dialog = document.querySelector('.education-dialog');
+
+          return listenForActions([
+            REQUEST_PATCH_USER_PROFILE,
+          ], () => {
+            helper.store.dispatch(requestPatchUserProfile(username));
+          }).then(() => {
+            let button = dialog.querySelector(".save-button");
+            assert(button.className.includes("disabled-with-spinner"));
+
+            TestUtils.Simulate.click(button);
+            assert.isFalse(patchUserProfileStub.called);
```

Comment:
Maybe check that there's an element with class `.mdl-spinner` here too? Not sure how important that is.